pyodide-build/pyodide_build/tests/test_common.py

```diff
@@ -113,3 +114,22 @@ def test_wheel_paths():
         "py3-none-any",
         "py2.py3-none-any",
     ]
+
+
+def test_search_pyodide_root():
```

Comment:
you can use the `tmp_path` fixture which directly returns a Path and auto-cleans at the end.